litellm/utils.py

```diff
@@ -4020,7 +4020,10 @@ def get_llm_provider(
                     or get_secret("TOGETHER_AI_TOKEN")
                 )
             elif custom_llm_provider == "friendliai":
-                api_base = "https://inference.friendli.ai/v1"
+                api_base = (
+                    get_secret("FRIENDLI_API_BASE")
```

Comment:
hey @pocca2048 this is missing support for a user dynamically passing in the api base - 

https://github.com/BerriAI/litellm/blob/72045cc0c5f27e170725f57636c87ac739dbc22b/litellm/utils.py#L3994

Reply:
Thank you for the suggestion! I have made changes.